app/views/recipes/index.html.erb

```diff
@@ -24,5 +24,10 @@
         <% end %>
       </tbody>
 </table>
-<%= link_to "New recipe", new_recipe_path %>
+<%= link_to "Detail recipe", new_recipe_path %>
+<% @recipes.each do |det|%>
+  <%= link_to recipe_path(det.id) do%>
+    <%= render det %>
+  <%end%>
```

Comment:
- [ ] Please delete these lines and make your recipe title clickable instead. Also style your recipes to match the given design: 

![image](https://user-images.githubusercontent.com/91486791/201355685-566afeab-5fde-4aaa-85bf-7b888f91de88.png)

Reply:
- [ ] Also kindly delete that "recipes branch" title as it is unnecessary.